docs/apim/4.1/getting-started/install-guides/installation-guide-migration.md

```diff
@@ -160,3 +160,11 @@ eventsCollection.find({"type": "PUBLISH_API"}).forEach((event) => {
        eventsCollection.replaceOne({ _id: event._id }, event);
 });
 ```
+
+## Introduction of USER_TOKEN Permission for Token Management
+
+Starting with version 4.1.26, a new permission called 'USER_TOKEN' has been introduced. This permission is specifically designed to control which users have the ability to read, create, update, and delete user tokens.
```

Comment:
I would change the opening sentence to: "Starting with version 4.1.26, There is a new permission called `USER_TOKEN`. 

For the 2nd sentence, I would just say "This permission controls which...."

Reply:
Should be ok now